modules/operations/pages/monitoring/new-relic.adoc

```diff
@@ -1,67 +1,69 @@
-= New Relic Integration
+= Integrate with New Relic
 
-[NOTE]
-====
-This article is a continuation of xref:monitoring/index.adoc[]. Please read that article first to understand the fundamentals of what resources are being used.
-====
+While there are multiple ways to https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic[send external Prometheus data to New Relic], one the https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#remote-write[Prometheus remote write integration] option is relevant to {product}.
```

Comment:
I think this should say: ...  **only** the Prometheus remote ...

**only** instead of **one**